package.json

```diff
@@ -0,0 +1,72 @@
+{
+  "name": "strong-config-ts",
+  "version": "0.0.1",
+  "description": "SOPS-based secret management tool for NodeJS",
```

Comment:
Nitty McNitterson: Official spelling is `Node.js` (according to on https://nodejs.org/en/)

Reply:
really

Reply:
yeah, i know. this will show up when searching our package on NPM and other places so i do believe the attention to detail demonstrates the amount of care that went into a product or package.

pretty please 🦄